src/android/InAppBrowser.java

```diff
@@ -773,6 +778,38 @@ public void onClick(View v) {
                 return _close;
             }
 
+            private void enableFullScreen(Activity activity, Window window) {
+                inFullScreen = true;
+                WindowManager.LayoutParams attrs = window.getAttributes();
+                attrs.flags |= WindowManager.LayoutParams.FLAG_FULLSCREEN;
+                attrs.flags |= WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON;
+                window.setAttributes(attrs);
+                if (android.os.Build.VERSION.SDK_INT >= 14)
```

Comment:
As of cordova-android@9, our [supported min SDK](https://github.com/apache/cordova-android/blob/0ce66249dafc916c5c2197d4c92cc51b555f777b/framework/cdv-gradle-config-defaults.json#L2) is 22.

We don't need to have defensive checks here, and can assume that the SDK will be at least 22.